core/object/object.h

```diff
@@ -556,6 +556,9 @@ class Object {
 		CONNECT_PERSIST = 2, // hint for scene to save this connection
 		CONNECT_ONE_SHOT = 4,
 		CONNECT_REFERENCE_COUNTED = 8,
+#ifdef TOOLS_ENABLED
```

Comment:
Guess no need for tools enabled since nothing really significant is added.

Reply:
Just wanted to make sure it didn't exist in release builds, although a good compiler would probably get rid of it, I'm sure...